Data/Browncoats.rte/Devices/Shields/Eruptor/Eruptor.lua

```diff
@@ -74,6 +77,7 @@ function ThreadedUpdate(self)
 				self.bashActive = false;
 				self.bashTimer:Reset();
 				self.bashCoolDown = true;
+				self.hitActor = nil;
```

Comment:
wouldn't it be safer to just set hitActor to nil at the very top of ThreadedUpdate?